package.json

```diff
@@ -187,7 +187,8 @@
             "mojo": "mojo run",
             "erlang": "escript",
             "spwn": "spwn build",
-            "pkl": "cd $dir && pkl eval -f yaml $fileName -o $fileNameWithoutExt.yaml"
+            "pkl": "cd $dir && pkl eval -f yaml $fileName -o $fileNameWithoutExt.yaml",
+            "gleam": "gleam run -m $fileNameWithoutExt"
```

Comment:
How is the folder structure? What if the file is not in the root? Will it still work?

Reply:
It only works for files on first level nesting: `src/*.gleam`
For deep nested files like: `src/**/*.gleam` it's not working
For able to work on deep nested level need to have some rare customized parameter like `$relativeFileNameWithoutExtToFolder("src")`

Reply:
It seems `src/*.gleam` is the pattern of gleam: https://gleam.run/writing-gleam/
So current way is OK to me.